gst/edge/edge_sink.c

```diff
@@ -0,0 +1,344 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2022 Samsung Electronics Co., Ltd.
+ *
+ * @file    edge_sink.h
+ * @date    01 Aug 2022
+ * @brief   Publish incoming streams
+ * @author  Yechan Choi <yechan9.choi@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "edge_sink.h"
+
+GST_DEBUG_CATEGORY_STATIC (gst_edgesink_debug);
+#define GST_CAT_DEFAULT gst_edgesink_debug
+
+/**
+ * @brief the capabilities of the inputs.
+ */
+static GstStaticPadTemplate sinktemplate = GST_STATIC_PAD_TEMPLATE ("sink",
+    GST_PAD_SINK,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS_ANY);
+
+/**
+ * @brief edgesink properties
+ */
+enum
+{
+  PROP_0,
+
+  PROP_HOST,
+  PROP_PORT,
+
```

Comment:
Let's add `PROP_CONNECT_TYPE` for later use.

Reply:
Updated

Reply:
Updated.